File: client/src/app/types/movie.ts
import { Actor } from "./actor";

export class Movie {
    name : string = '';
    year: number = 0;
    imgSrc : string = "";
    description : string = "";
    genre : string = "";
    rdfTurtle : string = "";
    url : string = "";
    actors : Actor[] = [];

    constructor(data?: any) {
        Object.assign(this, data);
    }
}

File: server/src/main/java/com/ucv/ace/types/Movie.java
package com.ucv.ace.types;

import lombok.Data;

import java.util.List;

@Data
public class Movie {

    private int year;

    private String name;

    private String url;

    private String imgSrc;

    private String rdfTurtle;

    private String description;

    private String genre;

    private List<Actor> actors;

}
